support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStepExecution.java

```diff
@@ -54,6 +58,7 @@ public boolean start() throws Exception {
         actions.add(new CauseAction(new Cause.UpstreamCause(invokingRun)));
         List<ParameterValue> parameters = step.getParameters();
         if (parameters != null) {
+            completeDefaultParameters(parameters, (Job) project);
```

Comment:
And what happens if there are no parameters in the call? Will core add all the default parameters for you?
I'm asking because I can't remember and at least it would be nice to have a test for it.